File: hash/1.java
import java.util.*;
class Solution {
    public String solution(String[] participant, String[] completion) {
        String answer = "";
        Arrays.sort(participant);
        Arrays.sort(completion);
        int i;
        for(i = 0; i < completion.length; i++){
            if(!participant[i].equals(completion[i])){
                answer = participant[i];
                    break;
            }
        }
        answer = participant[i];
        return answer;
    }
}
// java로 코테 연습이 필요해서 python으로 풀었던 문제를 다시 풀어 보았다 실제 풀이 방식은 python과 동일

File: stack/1.py
def solution(prices):
    answer = [0] * len(prices)

    for i in range(len(prices)-1):
        for j in range(i, len(prices)-1):
            if prices[i] >prices[j]:
                break
            else:
                answer[i] +=1
    return answer

#가격비교를 위해서 숫자를 비교하였다 비교해서 만약 자신보다 작은숫자가 나온순간 brak를 통해서 나오고 몇번만에 작은수가 나온지 체크한다
#마지막 숫자는 비교할 필요가 없기 때문에 마지막 전까지만 for문이 돌아가게 하였다